### PR TITLE
TPROD-357 - CoreBundle - Standard - Migrate PHP Templates to Twig Templates

### PR DESCRIPTION
**Pull request by JoshuaEstes**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.

### CODE REVIEW

**Comment by mabumusa1** [comment]
@escopecz Can you help us with testing this PR? According to @JoshuaEstes 

```CoreBundle - Standard // Ready, however I'm not sure what bundles or other templates use this. Looking at the code, it defaults to these templates unless the bundle contains the template file. I wasn't able to find anything that uses these templates.```

**Comment by mabumusa1** [comment]
@JoshuaEstes can you increase the coverage for this PR?

**Comment by escopecz** [comment]
~~I can see the "standard" controller actions and therefore templates be used in:~~
- campaigns
- marketing messages
- webhooks
- focus items
- tweets
~~So as long as you can edit/create/delete/view those entities and it looks the same as before then we are good to go.~~

Edit: I was wrong, the bundle specific templates are configured with the `getTemplateBase` method. I cannot find any place where the standard templates would be used.

**Comment by escopecz** [comment]
> @JoshuaEstes can you increase the coverage for this PR?

I wouldn't worry about this. The code is not adding any new lines in PHP files. The calculation of the coverage is wrong.

File: app/bundles/CoreBundle/Controller/AbstractStandardFormController.php [hunk 1]
```diff
@@ -727,14 +730,15 @@ protected function getTemplateName($file, string $engine = self::ENGINE_TWIG)
         $originalFile = $file;
         if (self::ENGINE_TWIG === $engine && strpos($file, '.php')) {
             $file = str_replace('.php', '.twig', $file);
+            trigger_deprecation('mautic', '5.x', 'Using PHP Templates is deprecated, use Twig Templates instead.');
```
Comment (JoshuaEstes) [inline app/bundles/CoreBundle/Controller/AbstractStandardFormController.php:733]: https://github.com/symfony/deprecation-contracts

I would like to keep this here in order to make sure that anything else using PHP templates will trigger a deprecation error to show up in the logs.

Once the PHP Template Engine is completely removed, this method will be rewritten.

Comment (escopecz) [reply app/bundles/CoreBundle/Controller/AbstractStandardFormController.php:733]: Good idea!

**Review by escopecz — APPROVED** [review]
Thanks! 👍

**Review by mabumusa1 — APPROVED** [review]
I did a code review this PR is good to go

**Comment by escopecz** [comment]
@JoshuaEstes so this did broke the Marketing Messages. We'll have to revert this and migrate the templates to Twig.

**Comment by escopecz** [comment]
Revert PR: https://github.com/mautic/mautic/pull/12021

**Pull request merged**